python/packages/autogen-ext/src/autogen_ext/agents/web_surfer/playwright_controller.py

```diff
@@ -1,9 +1,12 @@
 import asyncio
 import base64
+import io
 import os
 import random
 from typing import Any, Callable, Dict, Optional, Tuple, Union, cast
 
+# TODO: Fix unfollowed import
+from markitdown import MarkItDown  # type: ignore
```

Comment:
This is a pretty heavyweight dependency. I have mixed feelings about it including here, where it is used minimally (vs. other means of converting the DOM to text). Mabe we should make this optional (attempt the import in a try...catch), and backoff to something simpler if the dependency is not installed?

Reply:
added your suggestion, happy with both solutions